### PR TITLE
adds the "how to charge your PDA" contraband poster

### PR DESCRIPTION
**Pull request by capsaicinz**
## About The Pull Request

speaks for itself, it's called wasting my GBP

a

## Why It's Good For The Game

unknown

## Changelog
:cl:
add: adds the "how to charge your PDA" poster
/:cl:

### CODE REVIEW

**Comment by GoldenAlpharex** [comment]
Hahaha

**Comment by LemonInTheDark** [comment]
Local main forgets to test

**Comment by capsaicinz** [comment]
get out of my headget out of my headget out of my headget out of my headget out of my headget out of my headget out of my headget out of my head

File: code/game/objects/effects/contraband.dm [hunk 1]
```diff
@@ -515,6 +515,11 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/poster/contraband/random, 32)
 	desc = "A Syndicate-commissioned poster that uses Syndie Moth™ to tell the viewer to keep the nuclear authentication disk unsecured. \"Peace was never an option!\" No good employee would listen to this nonsense."
 	icon_state = "aspev_syndie"
 
+/obj/structure/sign/poster/contraband/microwave
+	name = "How To Charge Your PDA" // here you go kyler
```
Comment (Mothblocks) [inline code/game/objects/effects/contraband.dm:519]:
```suggestion
	name = "How To Charge Your PDA"
```

Comment (capsaicinz) [reply code/game/objects/effects/contraband.dm:519]: i hate you

**Comment by Serxule** [comment]
maybe make it a bit clearer and less textured, its a little messy

**Comment by zxaber** [comment]
Rather than a lightning bolt, a battery mid-charge would probably be easier to understand and easier to sprite.

**Review by Imaginos16 — APPROVED** [review]
Hilarious poster

**Pull request merged**